app/views/users/index.html.erb

```diff
@@ -1 +1,15 @@
-Here is a list of users
\ No newline at end of file
+<% if @users.empty? %>
+   <h1>No users</h1>
+<% else %>
+    <ul>
+      <% @users.each do |user| %>
+        <li style='display:flex;'> 
+          <img src=<% user.photo %> />
+          <div>
+            <h3><%= user.name %></h3>
+            <p>Number of Posts: <%= user.posts_counter %></p>
+          </div>
+        </li>
+      <% end %>  
+    </ul>
```

Comment:
- [ ] [REQUIRED] Great job @Trust-007 , however, let's ensure that all our designs match the wireframe given.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
<img width="194" alt="Screenshot 2023-03-01 at 23 32 38" src="https://user-images.githubusercontent.com/58692590/222280888-b7a8262f-dad4-4bf7-8474-dcf741b7fee7.png"> | <img width="415" alt="Screenshot 2023-03-01 at 23 26 11" src="https://user-images.githubusercontent.com/58692590/222279633-bc7be3fe-22be-4a98-9d52-d5ec4f53fc9a.png">

- [ ] [REQUIRED] Kindly ensure that Clicking on each user takes you to their show page as shown in the [wireframe](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_user_page.png).